tests/integration/test_script_runner.py

```diff
@@ -351,9 +361,205 @@ class IntegrationTestFixture:
         util_script="tests/test_utils.py",
         extra_backend_dependencies=BackendDependencies.SQLALCHEMY,
     ),
+    IntegrationTestFixture(
+        name="split_data_on_whole_table_postgres",
+        user_flow_script="tests/integration/db/test_sql_data_split_on_whole_table.py",
+        data_context_dir="tests/integration/fixtures/no_datasources/great_expectations",
+        data_dir="tests/test_sets/taxi_yellow_tripdata_samples/",
+        util_script="tests/test_utils.py",
+        other_files=(
+            (
+                "tests/integration/fixtures/split_and_sample_data/postgres_connection_string.yml",
+                "connection_string.yml",
+            ),
+        ),
+        extra_backend_dependencies=BackendDependencies.POSTGRESQL,
+    ),
+    IntegrationTestFixture(
```

Comment:
should go to appropriate `cloud_db_tests` list, and similar story for the rest of these `IntegrationTextFixture` objects